sdc/tests/test_series.py

```diff
@@ -4392,6 +4615,7 @@ def test_series_pct_change_impl(S, periods=1, fill_method='pad', limit=None, fre
     "test_series_op4",
     "test_series_op6",
     "test_series_op7",
+    "test_series_operator_add_numeric_same_index_default",
```

Comment:
This code is not in master after #340.

Reply:
Yes, thank you, I'll use new skip decorators.